systems-management/nisysmgmt.yml

```diff
@@ -0,0 +1,2030 @@
+              displayversion: 19.5.0
+              url: http://www.ni.com
+              packager: National Instruments <support@ni.com>
+              priority: standard
+              group: Infrastructure
+              version: 19.5.0.28-0+d28
+      feeds:
+        description: Feeds that are configured on the system.
+        $ref: '#/definitions/ManagedSystemDataProperty'
+        example:
+          last_updated_timestamp: 2019-02-19T11:42:25.078Z
+          data:
+            http://download.ni.com/support/nipkg/products/ni-package-manager/released:
+            - configurable: false
+              enabled: true
+              name: ni-package-manager-released
+              uri: http://download.ni.com/support/nipkg/products/ni-package-manager/released
+            http://download.ni.com/support/nipkg/products/ni-package-manager/eulas:
+            - configurable: false
+              enabled: true
+              name: ni-package-manager-eulas
+              uri: http://download.ni.com/support/nipkg/products/ni-package-manager/eulas
+      sysapi:
+        description: System API information.
+        $ref: '#/definitions/ManagedSystemDataProperty'
+        example:
+          last_updated_timestamp: 2019-02-19T11:42:25.078Z
+          data:
+            - resource_uri: nisyscfg//sys/bus/pci/devices/0000:04:00.0
+              properties:
+              - type: string
+                tag: 16777216
+                value: "//localhost/nisyscfg/%2Fsys%2Fbus%2Fpci%2Fdevices%2F0000:04:00.0"
+              - type: bool
+                tag: 16781312
+                value: true
+            - resource_uri: nisyscfg/enp0s25
+              properties:
+              - type: string
+                tag: 16777216
+                value: "//localhost/nisyscfg/enp0s25"
+              - type: bool
+                tag: 16781312
+                value: false
+      groups:
+        description: Groups information about a system. Specifies on which groups is the system assigned to.
+        $ref: '#/definitions/ManagedSystemDataProperty'
+        example:
+          last_updated_timestamp: 2019-02-19T11:42:25.078Z
+          data:
+            - 121b471c-d7d4-4996-a38f-27f7e58be070
+      keywords:
+        description: Keywords metadata information about a system.
+        type: object
+        properties:
+          last_updated_timestamp:
+            description: Last updated timestamp.
+            type: string
+            format: iso-date-time
+            example: 2019-02-19T11:42:25.078Z
+          data:
+            description: A list of strings defining metadata information about a system.
+            type: array
+            items:
+              type: string
+              example: 
+                - testVM
+        required: [last_updated_timestamp, data]
+      properties:
+        description: Properties metadata information about a system.
+        type: object
+        properties:
+          last_updated_timestamp:
```

Comment:
This 'last_updated_timestamp' needs an example element so that it does not render as "string".